### ISSUE TITLE
Allow pasting existing config to demo

### ISSUE THREAD

**Issue by kkmuffme** [eslint/eslint.github.io#574]
https://eslint.org/demo/ is a great tool to visually see the effect of rules/rule changes.

It already allows me to download the selected config, but it would be great if I could also paste an existing config from .eslintrc.json to use, so I don't always have to manually check the checkboxes.

**Comment by kaicataldo** [comment]
I agree that this would be a nice thing to have. When thinking about improvements we can make to the demo post site redesign, I’ve thought it could be nice to have both a configuration similar to how we do it now as well as seeing the actual generated JSON.
